### PR TITLE
Print out database error messages

### PR DESCRIPTION
**Pull request by annando**
This should help analysing database problems.

### CODE REVIEW

File: include/dbstructure.php [hunk 1]
```diff
@@ -158,6 +158,13 @@ function print_structure($database) {
 	}
 }
 
+function print_update_error($db, $message) {
```
Comment (rabuzarus) [inline include/dbstructure.php]: STANDARDS: please add a doxygen header

File: include/dbstructure.php [hunk 2]
```diff
@@ -158,6 +158,21 @@ function print_structure($database) {
...
+/**
+ * @brief Print out database error messages
+ *
+ * @param object $db Database object
+ * @param string $message Message to be added to the error message
+ *
+ * @return string Error message
+ */
+function print_update_error($db, $message) {
+	echo sprintf(t("\nError %d occured during database update:\n%s\n"),
```
Comment (Hypolite) [inline include/dbstructure.php]: Why did you add an `echo` here? Why not returning the DB error as well?

Comment (annando) [inline include/dbstructure.php]: It is meant for calling the script by hand.
